pkg/api/steve/disallow/disallow.go

```diff
@@ -9,8 +9,10 @@ import (
 	steve "github.com/rancher/steve/pkg/server"
 )
 
+// The resource names must be plural.
 var (
-	allowAll = map[string]bool{
+	// AllowAll is a set of resources for which Rancher doesn't require authentication to perform any operation.
+	AllowAll = map[string]bool{
```

Comment:
Not sure if we want this to be capitalized since none of the other maps are capitalized

Reply:
Unfortunately, we need to export this map, that's why it's capitalized. Not great to have exported package-level variables, but we need this to avoid a package redesign here and now.